portfolio/src/main/java/com/google/sps/servlets/NewServlet.java

```diff
@@ -1,19 +1,32 @@
 package com.google.sps.servlets;
 
 import java.io.IOException;
+import java.util.ArrayList;
 
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
+import com.google.gson.Gson;
 
-@WebServlet("/servlet")
-public class NewServlet extends HttpServlet {
 
+@WebServlet("/step_one_servlet")
+public class NewServlet extends HttpServlet {
+    String[] phrases = new String[] {"Today is a good day!","It works!","Hi"};
+    
```

Comment:
no need to have `= new String[]`, you can drop `new String[]`.

`String[] phrases = {"Today is a good day!","It works!","Hi"};`